specification/app/resource-manager/Microsoft.App/preview/2023-05-02-preview/examples/Patches_ListByBuilderResource.json

```diff
@@ -0,0 +1,74 @@
+{
+  "operationId": "Patches_ListByBuilderResource",
+  "parameters": {
+    "api-version": "2023-05-02-preview",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "rg",
+    "builderName": "testBuilder"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg/providers/Microsoft.App/builders/testBuilder/patches/testPatch-25fe4b",
+            "name": "testPatch-25fe4b",
+            "type": "Microsoft.App/builders/patches",
+            "properties": {
+              "provisioningState": "Succeeded",
+              "patchStatus": "NotStarted",
+              "patchDetails": {
+                "patchName": "testPatch-25fe4b",
+                "targetImageName": "testregistry.azurecr.io/test-image:latest",
+                "newRunImage": "mcr.microsoft.com/oryx/builder:run-dotnet-aspnet-7.0.7-cbl-mariner2.0",
+                "oldRunImage": "mcr.microsoft.com/oryx/builder:run-dotnet-aspnet-7.0.5-cbl-mariner2.0",
+                "reason": "Newer run image available.",
+                "targetResourceGroup": "rg",
+                "targetContainerAppEnv": "test-env",
+                "targetContainerApp": "test-app",
```

Comment:
How about targetContainerAppId?

Like environmentId in Builders. The Id contains rg, and app type, app name

Reply:
that makes a lot more sense. Thanks for the suggestion!